templates/deployment.yaml

```diff
@@ -138,6 +138,42 @@ spec:
             readOnly: true
           {{- end }} 
 
+        - name: {{ .Chart.Name }}-job-scanner-staging
```

Comment:
deployment and staging are two deployments of the chart, you do not need both inside the chart.